guides/request-validation-php-lumen/example-2/example-2.5.x.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Http\Request;
+use Twilio\Twiml;
+
+$app->post('/voice', ['middleware' => 'TwilioRequestValidator',
```

Comment:
Could you double check if it is needed to prepend a `/` to the route name please?